### PR TITLE
[v2.6] Release Epinio OOB

### PR DESCRIPTION
**Pull request by PennyScissors**
This PR is to release the Epinio chart OOB on behalf of the Epinio team.
The chart was validated by QA and approved here: https://github.com/rancher/charts/pull/1814#issuecomment-1262402359

### CODE REVIEW

File: release.yaml
Comment (aiyengar2) [inline release.yaml:4]: these should be deleted

Comment (PennyScissors) [reply release.yaml]: done

File: charts/epinio/100.0.0+up1.2.1/Chart.yaml
Comment (aiyengar2) [inline charts/epinio/100.0.0+up1.2.1/Chart.yaml:13]: is this correct?

**Review by aiyengar2 — APPROVED** [review]
From a packaging perspective, this PR seems good to me.

However, disclaimers:
- Since this is a copy-paste of a package introduced in dev-v2.6, I haven't reviewed the **content** of the chart in this PR since I assume that that review has already been done by another reviewer with this PR in dev-v2.6. For example, I'm not blocking my approval on a review to the comment I have left on this PR regarding the upstream-version since I assume this was discussed on merging the initial PR to dev-v2.6.
- I'm assuming that Epinio's QA has already validated the functionality of this chart.

Having made these disclaimers, I'm approving this PR. cc: @prachidamle @MKlimuszka let me know if I should merge or whether we need a deeper content review.

File: charts/epinio/100.0.0+up1.2.1/Chart.yaml
Comment (PennyScissors) [reply charts/epinio/100.0.0+up1.2.1/Chart.yaml:13]: That would be a question for the Epinio team. Maybe @andreas-kupries or @thehejik?

**Comment by MKlimuszka** [comment]
Tomas (QA) approved releasing it today on Slack

**Review by MKlimuszka — APPROVED** [review]

**Comment by prachidamle** [comment]
@aiyengar2 @PennyScissors we should be reviewing the chart w.r.t chart packaging, not validation of content /functionality the chart is providing.
@andreas-kupries validation of the chart functionality and content would be the responsibility of Epinio team, we will only review that the chart packaging is valid to release.

File: charts/epinio/100.0.0+up1.2.1/Chart.yaml
Comment (andreas-kupries) [reply charts/epinio/100.0.0+up1.2.1/Chart.yaml:13]: @PennyScissors 

If the 19.0.3 refers to the epinio chart version then this is wrong. Should be `1.2.1`.
See also `100.0.0+up1.2.1` in the file name. 

I see this bad version in the #1814 as well. I do not remember writing it, anymore. I suspect that this is a copy/paste error I did. I.e. that I copied the annotations from some example or existing chart and then missed editing this one to the correct version number.

Apologies for that.

As this (2126) is your PR I suspect that I will not be able to fix that, right ? (I.e. no permissions)

Comment (MKlimuszka) [reply charts/epinio/100.0.0+up1.2.1/Chart.yaml:13]: @andreas-kupries , I thought we were all good to release? I'm confused between this message and the Slack messages. Can we merge this as is?

Comment (andreas-kupries) [reply charts/epinio/100.0.0+up1.2.1/Chart.yaml:13]: I thought that we were good to release. I got very much blind-sided by the issue above.

If a bogus `upstream-version` like the above is a blocker (*) then we are not good to release, and should fix that annotation in the modified chart. Should (can?) this be fixed in this PR ? Or should I create a PR fixing it and then @PennyScissors updates this here with it ?

---
(*) I do not know what Rancher (marketplace) considers a blocker vs not.

Comment (andreas-kupries) [reply charts/epinio/100.0.0+up1.2.1/Chart.yaml:13]: To work ahead a bit, PR #2135 is a dev-v2.6 PR which fixes the annotation in the chart, and only that.
I hope that making it was the right thing to do.

Comment (aiyengar2) [reply charts/epinio/100.0.0+up1.2.1/Chart.yaml:13]: I think the call for whether the upstream version should be a blocker for release is up to the Epinio team! 

From the charts side, we have approved the PR and are just waiting for your approval on the PR review @andreas-kupries before merging.

**Comment by aiyengar2** [comment]
## Charts Checklist (built for v0.3.x charts-build-scripts)

Scripts Gist: https://gist.github.com/PennyScissors/70fac62fcbda2030415474bfea842143

### Checkpoint 0: Validate `release.yaml`

Validation steps:
- [x] Each chart version in `release.yaml` DOES NOT modify an already released chart. If so, stop and modify the versions so that it releases a net-new chart.
- [x] Each chart version in `release.yaml` IS exactly 1 more patch version than the last released chart version. If not, stop and modify the versions so that it releases a net-new chart.

### Checkpoint 1: Compare contents of assets/ to charts/

Validation steps:
- [x] Running `make unzip` to regenerate the `charts/` from scratch, then `git diff` to check differences between `assets/` and `charts/` yields NO differences or innocuous differences.

IMPORTANT: Do not undo these changes for future steps since we want to keep the charts/ that match the current contents of assets!

### Checkpoint 2: Compare assets against index.yaml

Validation steps:
- [x] The `index.yaml` file has an entry for each chart version.
- [x] The `index.yaml` entries for each chart matches the `Chart.yaml` for each chart.
- [x] Each chart has ALL required annotations
  - kube-version annotation
  - rancher-version annotation
  - permits-os annotation (indicates Windows and/or Linux)

### Checkpoint 3: Logical Checks

Omitted, as described in https://github.com/rancher/charts/pull/2126#issuecomment-1262749863

**Review by aiyengar2 — APPROVED** [review]

**Review by andreas-kupries — COMMENTED** [review]
Approved.

**Pull request merged**
